From 635f9ca8478e5e5d5cabdc6194a89379440fd54c Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 25 Jun 2018 14:43:36 +0800
Subject: [PATCH 4066/5242] MLK-18673 ARM64: dts: freescale: imx8dx: disable
 DRC thermal zone

commit  0d07f80f2a163c145b39e347bc4b77990fa39524 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enabling DRC thermal zone could impact DDR PLL clock quality
under certain conditions, disable it for now.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi |   18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index 79d42d5..8f4aca8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -1387,24 +1387,6 @@
 				};
 			};
 		};
-
-		drc-thermal0 {
-			polling-delay-passive = <250>;
-			polling-delay = <2000>;
-			thermal-sensors = <&tsens 1>;
-			trips {
-				drc_alert0: trip0 {
-					temperature = <85000>;
-					hysteresis = <2000>;
-					type = "passive";
-				};
-				drc_crit0: trip1 {
-					temperature = <127000>;
-					hysteresis = <2000>;
-					type = "critical";
-				};
-			};
-		};
 	};
 
 	irqsteer_csi: irqsteer@58220000 {
-- 
1.7.9.5

